### DISCUSSION TITLE
PrivacyTests.org result for Bromite compared to other browsers

### DISCUSSION THREAD

**Discussion by Raul1198** [bromite/bromite#2472] [General]
Recently i stumble upon privacytests.org where there were many test for different browser for different platform there was Bromite too . Bromite passed almost most of them but failed some, those failed test does they matter like below we see fingerprinting resistance tests (2of7) compared to Mull, tracking query parameter tests (0of24) compared to Brave.

![chrome_screenshot_23-Nov-2022 9_30_11 am IST](https://user-images.githubusercontent.com/29801315/203467815-5594fb1c-b4e7-4592-9094-f67897419c56.png)

**Comment by uazo** [comment]
> Bromite passed almost most of them but failed some

If you activate this [flag](https://github.com/bromite/bromite/issues/2337), all those in "State Partitioning tests" will turn green.
"Fingerprinting resistance tests" will be fixed with the [Viewport protection](https://github.com/bromite/bromite/pull/2294), but those points will never turn green because the tests are only made for the patch that the developer of that site made for brave.
the dots in "Tracking query parameter tests" will never be green because bromite has no intention of changing the user's navigation, but maybe in the future someone will propose some userscript that fixes it.

> compared to Mull

mull is from divestos right?
sooner or later I will contact him because I do not understand the basis on which he says protection is basic in bromite (ref https://divestos.org/index.php?page=browsers)

**Comment by mrx23dot** [comment]
What about rest of red flags?
